cogs/source/source.py

```diff
@@ -0,0 +1,146 @@
+from redbot.core import commands
+import tracemoepy
+from tracemoepy.errors import TooManyRequests
+from tracemoepy.errors import EntityTooLarge
+from tracemoepy.errors import ServerError
+from tracemoepy.errors import EmptyImage
+import math
+from discord import Embed
+
+
+class Source(commands.Cog):
+    """Looks up source of anime and pictures, may get NSFW results. Requires tracemoepy to be installed"""
+
+    """ run 'pip install tracemoepy' on your redbot virtual environment to install"""
+
+    # @commands.command()
+    @commands.group(name="source", invoke_without_command=True)
+    async def sourceCommand(self, ctx):
+        """
+        Looks for source of image
```

Comment:
nit: [PEP257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings)
```suggestion
        """Looks for source of image
```